test/helpers/position_assertions.cc

```diff
@@ -505,6 +505,8 @@ void UsageAssertion::check(const UnorderedMap<string, shared_ptr<core::File>> &s
                         // Should never happen.
                         ADD_FAILURE()
                             << "Error in test runner: identical locations weren't reported as subsets of one another.";
+                        expectedLocationsIt++;
```

Comment:
Fixes an infinite loop in the test runner that I encountered while testing this change.